core/asset/service.go

```diff
@@ -138,9 +138,14 @@ func (s *Service) GetLineage(ctx context.Context, urn string, query LineageQuery
 		return Lineage{}, fmt.Errorf("get lineage: get latest probes: %w", err)
 	}
 
+	assetDataAttributes, err := s.assetRepository.GetAssetDataAttributes(ctx, urns.list())
```

Comment:
I think this would need to be explicitly requested. Asset probes are very much within the lineage domain so it makes sense to return them by default.

Reply:
This was something that I was confused about how to make these changes... Will add a request param for this. 👍

Reply:
Given this - https://github.com/odpf/compass/pull/213#discussion_r1116556318

WDYT about the flag `with_node_props` ?